src/schema/schema.graphql

```diff
@@ -1406,9 +1406,14 @@ input CalendarItemMessageInput {
 	replyType: InviteReplyType #rt
 }
 
+input NewAttachmentInput {
+	attachmentId: ID
+}
+
 input SendMessageInput {
 	id: ID #id
 	origId: ID #origid
+	attach: [NewAttachmentInput]
```

Comment:
we already have AttachmentInput for this

Reply:
That is different and used for other purpose. We can't use that.

Reply:
I think it's same, it has extra key which can be ignored